mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = async (req, res) => {
+  try {
+    const deletionResult = await Order.findOneAndDelete({ _id: req.params.orderId });
+    if (!deletionResult) {
+      return res.status(404).json({ error: "Order not found" });
+    }
+    res.json({ message: "Order removed successfully" });
+  } catch (error) {
+    res.status(500).json({ error: errorHandler(error) });
```

Comment:
The error response object should have a consistent structure throughout the application. In this case, the error response object has a single property 'error' which is not consistent with other error responses in the codebase, such as in the `orderById` function where the error response object has a single property 'error' with a nested object. It would be better to maintain a consistent structure for error responses.